kie-spring-boot/pom.xml

```diff
@@ -44,24 +42,6 @@
 
   <dependencyManagement>
     <dependencies>
-      <dependency>
-        <groupId>org.springframework.boot</groupId>
-        <artifactId>spring-boot-dependencies</artifactId>
-        <version>${version.org.spring-boot}</version>
-        <type>pom</type>
-        <scope>import</scope>
-      </dependency>
-      <dependency>
-	<groupId>org.springframework.boot</groupId>
-	<artifactId>spring-boot-starter</artifactId>
-	<version>${version.org.spring-boot}</version>
-	<exclusions>
-	    <exclusion>
-	      <groupId>org.apache.logging.log4j</groupId>
-              <artifactId>log4j-api</artifactId>
-            </exclusion>
-	  </exclusions>
-      </dependency>
```

Comment:
I am a little bit worried about this exclusion which is not materialized here currently. Maybe we should leave it here as it was.

Reply:
Good point.